tasks/main_services.yml

```diff
@@ -60,39 +55,42 @@
       - "{{ volume_names.static }}:/static"
       - "{{ jwt_key_path }}:{{ jwt_key_path }}"
     # yamllint disable rule:line-length
-    env: "{{ base_server_environment_variables | combine(application_server_specific_environment_variables) | combine(hub_configuration.extra_environment_variables) }}" # noqa 204
+    env: "{{ base_server_environment_variables | combine(hub_configuration.extra_environment_variables) }}" # noqa 204
     # yamllint enable rule:line-length
-    command: supervisord -n -c /etc/supervisor/conf.d/supervisord.django.conf
+    labels: {
+      traefik.enable: "true",
+      traefik.http.routers.web.rule: "Host(`{{ hub_configuration.primary_hostname }}`)",
+      traefik.http.routers.web.entrypoints: "traefik_proxy-secured",
+      traefik.http.routers.web.tls.certresolver: "mytlschallenge",
+
+    # redirect secondary domain to primary
+    # Get Options
+      traefik.http.routers.web-secondary.rule: "Host(`{{ hub_configuration.secondary_hostnames | join('`,`') }}`) && Method(`GET`,`OPTIONS`)",
```

Comment:
don't understand why `GET` and `OPTIONS`. According to the working setup for volkswagen, it's `GET` and `HEAD`, everything else cannot be redirected.